aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -202,6 +203,52 @@ Tensor rsub(const Tensor& self, Scalar other, Scalar alpha) {
   return native::rsub(self, wrapped_scalar_tensor(other), alpha);
 }
 
+Tensor& bitwise_xor_out(Tensor& result, const Tensor& self, const Tensor& other) {
+  auto iter = TensorIterator::binary_op(result, self, other,
+    /*check_mem_overlap=*/true);
+  bitwise_xor_stub(iter.device_type(), iter);
+  return result;
+}
+
+Tensor bitwise_xor(const Tensor& self, const Tensor& other) {
+  Tensor result = at::empty({0}, self.options());
+  at::bitwise_xor_out(result, self, other);
+  return result;
+}
+
+Tensor& bitwise_xor_(Tensor& self, const Tensor& other) {
+  return at::bitwise_xor_out(self, self, other);
+}
+
+Tensor& bitwise_xor_out(Tensor& result, const Tensor& self, Scalar other) {
+  return at::bitwise_xor_out(result, self, wrapped_scalar_tensor(other));
```

Comment:
I'm curious the perf impact of dispatching `scalar` to tensor version of functions. 
btw the failed XLA is not only XLA device, if you add a cpp test for `xor` with `Scalar`, it'll likely fail as well. (I tried commenting out the XLA part of test and it failed). 
```
    TEST_F(AtenXlaTensorTest, TestBitwiseXorScalar) {
      torch::Tensor lhs = torch::randint(0, std::numeric_limits<int32_t>::max(),
                                         {4, 2}, torch::TensorOptions(torch::kInt));
      torch::Scalar rhs(123456789);
      torch::Tensor result = lhs.__xor__(rhs);
```

Reply:
I have pushed an update -- this is indeed broken after replace `native::` with `at::`. The cause seems to be that the tensor returned by `wrapped_scalar_tensor` cannot survive some kinds of `at::` functions.